.gitignore

```diff
@@ -1,2 +1,4 @@
 target
 Cargo.lock
+.idea
+cucumber.iml
```

Comment:
In a past life, I was told that the project gitignore should only contain things specific to the project to be ignored, and that developers should put their editor stuff in their personal global gitignore.

I actually dont really mind either way though, no change here.